apps/backend/src/parse/parse.service.ts

```diff
@@ -248,7 +239,7 @@ export class ParseService {
    * @returns List of uploaded files
    */
   async getAllFiles(): Promise<FileUploadedEntity[]> {
-    return this.uploadedRepo.find();
+    return await this.uploadedRepo.find();
```

Comment:
Unnecessary, I'm pretty sure. Since `repo.find` is an async function, the await goes to the top.

Same with the ones below (saveFileUploaded)